suricata.yaml.in

```diff
@@ -1474,6 +1474,13 @@ flow:
   #managers: 1 # default to one flow manager
   #recyclers: 1 # default to one flow recycler thread
 
+# Mark a flow as an elephant flow if the rate of influx of bytes is equal to
+# or exceeds the given number. The rate is calculated by dividing the total
+# number of bytes a flow has seen in either direction divided by the time a
```

Comment:
s/either/both

Reply:
time in seconds ?

Reply:
yes. That's important to mention. I missed it.